src/Streamly/Internal/Data/Unfold/Type.hs

```diff
@@ -125,7 +125,7 @@ yieldM m = Unfold step inject
 
 -- | Discards the unfold input and always returns the argument of 'yield'.
 yield :: Applicative m => b -> Unfold m a b
-yield = yieldM . pure
+yield = yieldM Prelude.. pure
```

Comment:
`Prelude..` looks pretty weird. `.` is a very important operator and can be imported normally (unqualified).

Reply:
Its because when we define the Category instance it becomes ambiguous.